osu.Game/OsuGame.cs

```diff
@@ -1428,9 +1428,18 @@ private T loadComponentSingleFile<T>(T component, Action<Drawable> loadCompleteA
 
         public bool OnPressed(KeyBindingPressEvent<GlobalAction> e)
         {
+            switch (e.Action)
+            {
+                case GlobalAction.DecreaseVolume:
+                case GlobalAction.IncreaseVolume:
+                    return volume.Adjust(e.Action);
```

Comment:
why is this block duplicated now? another copy of it is still there lower down

https://github.com/ppy/osu/blob/e752531aec5dea9401b55afc312c8f625673dba6/osu.Game/OsuGame.cs#L1447-L1449

Reply:
Oversight from lack of self checks for sure